### DISCUSSION TITLE
Local backup decrypted without files exporting (v1.5.4)

### DISCUSSION THREAD

**Discussion by sky-and-sand** [mvt-project/mvt#264] [Q&A]
I am testing a local backup through Finder of a mobile device. Successfully decrypted a folder/backup but after MVT exported output to my dir of choice, the directory is empty?

Success with decrypt-backup -d
then
Failed on check-backup --output 

several "unable to extract device information"
from the folder where decrypted was saved. 

Following Documentation for this step:
Run mvt-ios on a Backup
https://docs.mvt.re/en/latest/ios/backup/check/

thanks

**Comment by Te-k** [comment]
Hi, could you share an extract of the console output?
